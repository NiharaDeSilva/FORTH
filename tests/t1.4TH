PrintingStack!!! .
20 10 5