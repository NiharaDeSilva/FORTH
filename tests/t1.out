Stack is not empty at the end of execution
Stack content:[Integer 5,Integer 10,Integer 20]